### PR TITLE
new icon: putty (original)

### PR DESCRIPTION
**Pull request by MichielPater**

### CODE REVIEW

File: icons/putty/putty-original.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...H19.84zm0 0" class="Q"/><path d="M17.074 71.371h41.395l-.922 1.844H18.918V104.5l-1.844 1.844zm0 0" class="R"/><path d="M18.918 99.891h.922v.922h-.922zm0 0" class="S"/><g class="T"><path d="M5.121 108.16l9.188-9.191h.922v1.844h-.922l-7.371 7.375v12.875H67.66l7.375-7.371v-12.879h-8.27l1.844-1.844h8.266v14.723l-9.187 9.188H5.121zm0 0"/><path d="M17.074 71.371v34.973l-1.844 1.844V71.371l8.27-8.266h45.109v36.813l-8.27 8.27H15.23l1.844-1.844h43.238l6.453-6.453V64.922H23.527zm0 0"/><path d="M51.125 50.203l9.188-9.191h.922v1.844h-.922l-7.371 7.371v12.879h60.723l7.371-7.375V42.855h-8.266l1.84-1.844h8.27V55.73l-9.187 9.191H51.125zm0 0"/><path d="M63.078 13.391v34.969l-1.844 1.844V13.391l8.297-8.27h45.078v36.813l-8.266 8.27H61.234l1.844-1.844h43.238l6.453-6.453V6.965H69.531zm0 0"/></g><path d="M46.516 60.313h15.641L37.324 85.145v1.844h2.766l40.473-32.203v-1.844H65.844l23.91-24.832v-1.844H87.91L46.516 58.469zm0 0" fill="rgb(100%,100%,0.392157%)"/><path d="M40.398 87.91h-3.996v-3.148l23.555-23.551H45.621v-3.203l41.98-32.637h3.074v3.121L67.992 52.043h13.469v3.203zm-2.152-1.816h1.512l39.883-31.719v-.488H63.668L88.832 27.75v-.562h-.613L47.438 58.93v.461h16.945L38.246 85.531zm0 0" class="T"/></svg>
```
Comment (Panquesito7) [inline icons/putty/putty-original.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128" xmlns:v="https://vecta.io/nano"><style><![CDATA[.B{fill:rgb(1.568627%,1.176471%,100%)}.C{fill:rgb(3.137255%,3.137255%,74.117647%)}.D{fill:rgb(0%,1.176471%,45.098039%)}.E{fill:rgb(0%,1.568627%,45.098039%)}.F{fill:rgb(100%,100%,99.215686%)}.G{fill:rgb(74.901961%,74.901961%,75.294118%)}.H{fill:rgb(73.72549%,74.901961%,69.411765%)}.I{fill:rgb(76.078431%,76.078431%,76.078431%)}.J{fill:rgb(49.411765%,49.411765%,49.411765%)}.K{fill:rgb(75.294118%,75.294118%,75.294118%)}.L{fill:rgb(0.392157%,0.392157%,0.392157%)}.M{fill:rgb(50.196078%,50.196078%,50.196078%)}.N{fill:rgb(74.901961%,75.686275%,73.333333%)}.O{fill:rgb(77.647059%,73.72549%,74.901961%)}.P{fill:rgb(50.588235%,49.803922%,52.941176%)}.Q{fill:rgb(99.607843%,100%,98.039216%)}.R{fill:rgb(100%,100%,100%)}.S{fill:rgb(74.509804%,74.509804%,74.509804%)}.T{fill:rgb(0%,0%,0%)}]]></style><path d="M69.531 19.84h31.281v20.25H69.531zm0 0" class="B"/><path d="M69.531 18.918v22.094h-1.895l.051-22.094zm0 0" class="C"/><path d="M69.531 19.84h31.281v-1.844H67.688v1.844zm0 0" class="D"/><path d="M66.766 17.074h34.969v23.91H78.746v-.922h22.094V17.996H67.688v22.988l-.922.922zm0 0" class="E"/><path d="M23.527 77.797h31.281v20.25H23.527zm0 0" class="B"/><path d="M23.527 76.875v22.094h-1.895l.051-22.094zm0 0" class="C"/><path d="M23.527 77.797h31.281v-1.84H21.684v1.84zm0 0" class="D"/><path d="M54.809 98.047v-22.09H21.684v22.988h-.922v-23.91H55.73v23.91h-.922" class="E"/><path d="M23.527 98.047h31.281v.922H23.527zm0 0" fill="rgb(0%,0.392157%,87.45098%)"/><path d="M6.965 117.352v-9.191h56.113l7.375-7.371h4.605L64 111.848H10.648v5.527H6.965zm0 0" class="F"/><path d="M6.965 108.16l8.266-8.27v8.27zm0 0" class="G"/><path d="M6.965 117.352h3.684v3.684H6.965zm0 0" class="H"/><path d="M63.078 108.16l7.375-7.371h-2.766l-7.375 7.371zm0 0" class="I"/><path d="M75.035 100.813v12.879l-7.375 7.371H10.648v-3.687H64v-5.527zm0 0" class="J"/><path d="M41.934 114.609v2.766H10.648v-5.527H64v5.527h-3.687v-2.766zm0 0" class="K"/><path d="M41.934 114.609H60.34v2.766H41.934zm0 0" class="L"/><path d="M52.965 59.391v-9.187h56.117l7.371-7.375h4.609l-11.035 11.035H56.652v5.527zm0 0" class="F"/><path d="M52.965 50.203l8.27-8.27v8.27zm0 0" class="G"/><path d="M52.965 59.391h3.688v3.688h-3.687zm0 0" class="H"/><path d="M109.082 50.203l7.371-7.375h-2.762l-7.375 7.375zm0 0" class="I"/><path d="M121.035 42.828v12.879l-7.371 7.371H56.652v-3.687h53.352v-5.527zm0 0" class="J"/><path d="M87.91 56.652v2.738H56.652v-5.527h53.352v5.527h-3.687v-2.762H87.91zm0 0" class="K"/><path d="M87.91 56.652h18.406v2.738H87.91zm0 0" class="L"/><path d="M66.766 46.516h36.813V15.23l9.191-8.266v34.969l-6.453 6.449H64.922" class="M"/><path d="M63.078 13.391l40.473 1.84-1.844 1.844 4.609-3.684 6.453-6.453H69.531zm0 0" class="N"/><path d="M66.766 42.828h36.813v3.688H66.766zm0 0" class="O"/><path d="M64.922 41.934h.922l.922-.922V17.074h34.969l1.844-1.844H64.922zm0 0" class="P"/><path d="M65.844 41.934l.922-.922h34.969V17.074l1.844-1.844v27.598H65.844zm0 0" class="Q"/><path d="M63.078 13.391h41.395l-.922 1.84H64.922v28.52l-1.844.922zm0 0" class="R"/><path d="M64.922 41.934h.922v.922l-.922.922zm0 0" class="S"/><path d="M18.918 104.473h38.629V73.191l9.191-8.27v34.969l-6.449 6.453H17.074" class="M"/><path d="M17.074 71.371l40.473 1.844-1.84 1.844 4.605-3.687 6.453-6.449H23.527zm0 0" class="N"/><path d="M57.547 100.813v3.688H18.918v-4.609h.922v.922zm0 0" class="O"/><path d="M18.918 99.891h.922l.922-.922v-23.91H55.73l1.844-1.844H18.918zm0 0" class="P"/><path d="M19.84 99.891l.922-.922H55.73v-23.91l1.844-1.844v27.598H19.84zm0 0" class="Q"/><path d="M17.074 71.371h41.395l-.922 1.844H18.918V104.5l-1.844 1.844zm0 0" class="R"/><path d="M18.918 99.891h.922v.922h-.922zm0 0" class="S"/><g class="T"><path d="M5.121 108.16l9.188-9.191h.922v1.844h-.922l-7.371 7.375v12.875H67.66l7.375-7.371v-12.879h-8.27l1.844-1.844h8.266v14.723l-9.187 9.188H5.121zm0 0"/><path d="M17.074 71.371v34.973l-1.844 1.844V71.371l8.27-8.266h45.109v36.813l-8.27 8.27H15.23l1.844-1.844h43.238l6.453-6.453V64.922H23.527zm0 0"/><path d="M51.125 50.203l9.188-9.191h.922v1.844h-.922l-7.371 7.371v12.879h60.723l7.371-7.375V42.855h-8.266l1.84-1.844h8.27V55.73l-9.187 9.191H51.125zm0 0"/><path d="M63.078 13.391v34.969l-1.844 1.844V13.391l8.297-8.27h45.078v36.813l-8.266 8.27H61.234l1.844-1.844h43.238l6.453-6.453V6.965H69.531zm0 0"/></g><path d="M46.516 60.313h15.641L37.324 85.145v1.844h2.766l40.473-32.203v-1.844H65.844l23.91-24.832v-1.844H87.91L46.516 58.469zm0 0" fill="rgb(100%,100%,0.392157%)"/><path d="M40.398 87.91h-3.996v-3.148l23.555-23.551H45.621v-3.203l41.98-32.637h3.074v3.121L67.992 52.043h13.469v3.203zm-2.152-1.816h1.512l39.883-31.719v-.488H63.668L88.832 27.75v-.562h-.613L47.438 58.93v.461h16.945L38.246 85.531zm0 0" class="T"/></svg>

```

File: devicon.json [hunk 2]
```diff
@@ -2981,6 +2981,23 @@
...
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "original"
+            ]
+        },
+        "color": "#0000fc",
+        "aliases": []
```
Comment (Panquesito7) [inline devicon.json:3000]:
```suggestion
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            }
        ]
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by Thomas-Boi** [comment]
Hey @MichielPater,

Please hold off fixing the `fill` issue reported by the bot. I'm trying to find a way to fix this issue permanently using our SVG-optimizer bot.

File: icons/putty/putty-original.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...
```
Comment (Panquesito7) [inline icons/putty/putty-original.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128" width="128" height="128"><path d="M69.531 19.84h31.281v20.25H69.531zm0 0" fill="#0403ff"/><path d="M69.531 18.918v22.094h-1.895l.051-22.094zm0 0" fill="#0808bd"/><path d="M69.531 19.84h31.281v-1.844H67.688v1.844zm0 0" fill="#000373"/><path d="M66.766 17.074h34.969v23.91H78.746v-.922h22.094V17.996H67.688v22.988l-.922.922zm0 0" fill="#000473"/><path d="M23.527 77.797h31.281v20.25H23.527zm0 0" fill="#0403ff"/><path d="M23.527 76.875v22.094h-1.895l.051-22.094zm0 0" fill="#0808bd"/><path d="M23.527 77.797h31.281v-1.84H21.684v1.84zm0 0" fill="#000373"/><path d="M54.809 98.047v-22.09H21.684v22.988h-.922v-23.91H55.73v23.91h-.922" fill="#000473"/><path d="M23.527 98.047h31.281v.922H23.527zm0 0" fill="#0001DF"/><path d="M6.965 117.352v-9.191h56.113l7.375-7.371h4.605L64 111.848H10.648v5.527H6.965zm0 0" fill="#fffffd"/><path d="m6.965 108.16 8.266-8.27v8.27zm0 0" fill="#bfbfc0"/><path d="M6.965 117.352h3.684v3.684H6.965zm0 0" fill="#bcbfb1"/><path d="m63.078 108.16 7.375-7.371h-2.766l-7.375 7.371zm0 0" fill="#c2c2c2"/><path d="M75.035 100.813v12.879l-7.375 7.371H10.648v-3.687H64v-5.527zm0 0" fill="#7e7e7e"/><path d="M41.934 114.609v2.766H10.648v-5.527H64v5.527h-3.687v-2.766zm0 0" fill="silver"/><path d="M41.934 114.609H60.34v2.766H41.934zm0 0" fill="#010101"/><path d="M52.965 59.391v-9.187h56.117l7.371-7.375h4.609l-11.035 11.035H56.652v5.527zm0 0" fill="#fffffd"/><path d="m52.965 50.203 8.27-8.27v8.27zm0 0" fill="#bfbfc0"/><path d="M52.965 59.391h3.688v3.688h-3.687zm0 0" fill="#bcbfb1"/><path d="m109.082 50.203 7.371-7.375h-2.762l-7.375 7.375zm0 0" fill="#c2c2c2"/><path d="M121.035 42.828v12.879l-7.371 7.371H56.652v-3.687h53.352v-5.527zm0 0" fill="#7e7e7e"/><path d="M87.91 56.652v2.738H56.652v-5.527h53.352v5.527h-3.687v-2.762H87.91zm0 0" fill="silver"/><path d="M87.91 56.652h18.406v2.738H87.91zm0 0" fill="#010101"/><path d="M66.766 46.516h36.813V15.23l9.191-8.266v34.969l-6.453 6.449H64.922" fill="gray"/><path d="m63.078 13.391 40.473 1.84-1.844 1.844 4.609-3.684 6.453-6.453H69.531zm0 0" fill="#bfc1bb"/><path d="M66.766 42.828h36.813v3.688H66.766zm0 0" fill="#c6bcbf"/><path d="M64.922 41.934h.922l.922-.922V17.074h34.969l1.844-1.844H64.922zm0 0" fill="#817f87"/><path d="m65.844 41.934.922-.922h34.969V17.074l1.844-1.844v27.598H65.844zm0 0" fill="#fefffa"/><path d="M63.078 13.391h41.395l-.922 1.84H64.922v28.52l-1.844.922zm0 0" fill="#fff"/><path d="M64.922 41.934h.922v.922l-.922.922zm0 0" fill="#bebebe"/><path d="M18.918 104.473h38.629V73.191l9.191-8.27V99.89l-6.449 6.453H17.074" fill="gray"/><path d="m17.074 71.371 40.473 1.844-1.84 1.844 4.605-3.687 6.453-6.449H23.527zm0 0" fill="#bfc1bb"/><path d="M57.547 100.813v3.688H18.918v-4.609h.922v.922zm0 0" fill="#c6bcbf"/><path d="M18.918 99.891h.922l.922-.922v-23.91H55.73l1.844-1.844H18.918zm0 0" fill="#817f87"/><path d="m19.84 99.891.922-.922H55.73v-23.91l1.844-1.844v27.598H19.84zm0 0" fill="#fefffa"/><path d="M17.074 71.371h41.395l-.922 1.844H18.918V104.5l-1.844 1.844zm0 0" fill="#fff"/><path d="M18.918 99.891h.922v.922h-.922zm0 0" fill="#bebebe"/><path d="m5.121 108.16 9.188-9.191h.922v1.844h-.922l-7.371 7.375v12.875H67.66l7.375-7.371v-12.879h-8.27l1.844-1.844h8.266v14.723l-9.187 9.188H5.121zm0 0"/><path d="M17.074 71.371v34.973l-1.844 1.844V71.371l8.27-8.266h45.109v36.813l-8.27 8.27H15.23l1.844-1.844h43.238l6.453-6.453V64.922H23.527zm0 0"/><path d="m51.125 50.203 9.188-9.191h.922v1.844h-.922l-7.371 7.371v12.879h60.723l7.371-7.375V42.855h-8.266l1.84-1.844h8.27V55.73l-9.187 9.191H51.125zm0 0"/><path d="M63.078 13.391V48.36l-1.844 1.844V13.391l8.297-8.27h45.078v36.813l-8.266 8.27H61.234l1.844-1.844h43.238l6.453-6.453V6.965H69.531zm0 0"/><path d="M46.516 60.313h15.641L37.324 85.145v1.844h2.766l40.473-32.203v-1.844H65.844l23.91-24.832v-1.844H87.91L46.516 58.469zm0 0" fill="#FFFF01"/><path d="M40.398 87.91h-3.996v-3.148l23.555-23.551H45.621v-3.203l41.98-32.637h3.074v3.121L67.992 52.043h13.469v3.203zm-2.152-1.816h1.512l39.883-31.719v-.488H63.668L88.832 27.75v-.562h-.613L47.438 58.93v.461h16.945l-26.137 26.14zm0 0"/></svg>

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — COMMENTED** [review]
Hmm, this doesn't look so good. Could you please try improving the icon to meet the requirements as shown by the Peek Bot? The only idea I have is to make those parts transparent so it looks good. @Thomas-Boi, any other ideas? Thanks. 🙂

**Comment by Thomas-Boi** [comment]
I agree with @Panquesito7. The `original` version is way too complicated to be used as a plain version. Now, how do we make it into a plain version is a bit difficult. 

I'd recommend you making a line version containing the outlines of the computers and the lightning bolt. I don't know how to make a `plain` version that suits this specific logo unfortunately. If you have any ideas, feel free to push them here and we can run it through the bot.

If you are unwilling to modify the logo, we'll have to close this PR. While the quality is great, if it can't be made into an icon, it can't fit in this repo.

**Comment by MichielPater** [comment]
@Thomas-Boi @Panquesito7 added line version and plain version 😀

File: icons/putty/putty-line.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128" xmlns:v="https://vecta.io/nano"><path d="M60.313 41.012l-9.187 9.191v5.852l1.816-1.412v-4.416l7.371-7.371h.922v-1.844h-.922zm54.297 0l-1.84 1.844h8.266V55.73l-7.371 7.375H68.609v.043l-2.23 1.773h47.314l9.188-9.191V41.012h-8.271zM51.125 59.387v5.535h7.727l1.816-1.816h-7.727v-3.719h-1.816z"/><path d="M69.531 5.121l-8.297 8.27v34.801l1.844-1.436V13.391l6.453-6.426h43.238v34.941l-6.453 6.453H68.99l-1.775 1.844h39.129l8.266-8.27V5.121H69.531zM5.121 108.16l9.188-9.191h.922v1.844h-.922l-7.371 7.375v12.875H67.66l7.375-7.371v-12.879h-8.27l1.844-1.844h8.266v14.723l-9.187 9.188H5.121zm0 0"/><path d="M23.5 63.105l-8.27 8.266v36.816H60.34l8.27-8.27v-36.77l-2.23 1.773h.387v34.969l-6.453 6.453H17.074V71.371l6.453-6.449h35.324l1.816-1.816H23.5z"/><a><path d="M40.398 87.91h-3.996v-3.148l23.555-23.551H45.621v-3.203l41.98-32.637h3.074v3.121L67.992 52.043h13.469v3.203zm-2.152-1.816h1.512l39.883-31.719v-.488H63.668L88.832 27.75v-.562h-.613L47.438 58.93v.461h16.945l-26.137 26.14zm0 0"/></a></svg>
```
Comment (Panquesito7) [inline icons/putty/putty-line.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128" xmlns:v="https://vecta.io/nano"><path d="M60.313 41.012l-9.187 9.191v5.852l1.816-1.412v-4.416l7.371-7.371h.922v-1.844h-.922zm54.297 0l-1.84 1.844h8.266V55.73l-7.371 7.375H68.609v.043l-2.23 1.773h47.314l9.188-9.191V41.012h-8.271zM51.125 59.387v5.535h7.727l1.816-1.816h-7.727v-3.719h-1.816z"/><path d="M69.531 5.121l-8.297 8.27v34.801l1.844-1.436V13.391l6.453-6.426h43.238v34.941l-6.453 6.453H68.99l-1.775 1.844h39.129l8.266-8.27V5.121H69.531zM5.121 108.16l9.188-9.191h.922v1.844h-.922l-7.371 7.375v12.875H67.66l7.375-7.371v-12.879h-8.27l1.844-1.844h8.266v14.723l-9.187 9.188H5.121zm0 0"/><path d="M23.5 63.105l-8.27 8.266v36.816H60.34l8.27-8.27v-36.77l-2.23 1.773h.387v34.969l-6.453 6.453H17.074V71.371l6.453-6.449h35.324l1.816-1.816H23.5z"/><a><path d="M40.398 87.91h-3.996v-3.148l23.555-23.551H45.621v-3.203l41.98-32.637h3.074v3.121L67.992 52.043h13.469v3.203zm-2.152-1.816h1.512l39.883-31.719v-.488H63.668L88.832 27.75v-.562h-.613L47.438 58.93v.461h16.945l-26.137 26.14zm0 0"/></a></svg>

```

File: icons/putty/putty-plain.svg [hunk 5]
```diff
@@ -0,0 +1 @@
...844l23.91-24.832v-1.844H87.91zM14.684 101l-4 3.691L7 108.383v6.309V121h30.309H67.62l3.689-3.684L75 113.633v-6.316V101l-3.562.004-3.564.002-3.65 3.623-3.652 3.621H37.785 15v-3.625c0-2.752-.076-3.625-.316-3.625zm2.566-12.191V71.368l3.191-3.184L23.632 65h16.12 16.12L49.5 71.375l-6.371 6.375h-11.19H20.75V88v10.25h17.125H55V88 77.75h-.79c-.703 0-.056-.583 5.813-5.244l6.603-5.244.064 16.176.064 16.176-3.311 3.318-3.311 3.318H38.691 17.25zm55-25.912c0-.057 2.081-1.765 4.625-3.795L81.5 55.41v-1.705V52H75c-3.575 0-6.5-.091-6.5-.203s.316-.506.701-.875l.701-.672h18.365 18.365l3.619-3.625L113.869 43h3.566H121v6.316 6.316l-3.691 3.684-3.69 3.684H92.934c-11.376 0-20.684-.046-20.684-.103zm-.375-14.716c0-.038 1.73-1.864 3.844-4.059l3.844-3.99 10.656-.004 10.656-.004V29.938 19.75H83.813 66.75v10.182 10.182l.844.037.844.037-2.625 2.029-2.625 2.029-.032-15.403-.032-15.403 3.218-3.22L69.561 7h21.595 21.595v17.407 17.407l-3.22 3.218-3.22 3.218H89.093c-9.47 0-17.218-.031-17.218-.069zm-18.842 2.95v-.841l3.67-3.667c2.63-2.627 3.784-3.667 4.073-3.667.391 0 .402.043.352 1.45l-.051 1.45-3.889 3.048-4.022 3.057c-.073.005-.133-.369-.133-.831zm0 11.094v-.884h3.308 3.308l-.871.884-.871.884H55.47h-2.437z"/></svg>
```
Comment (Panquesito7) [inline icons/putty/putty-plain.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128" height="128" viewBox="0 0 128 128" fill="#bfc1bb" xmlns:v="https://vecta.io/nano"><path d="M87.91 26.266L46.516 58.469v1.844h15.641L37.324 85.145v1.844h2.766l40.471-32.201.002-.082v-1.764H65.844l23.91-24.832v-1.844H87.91zM14.684 101l-4 3.691L7 108.383v6.309V121h30.309H67.62l3.689-3.684L75 113.633v-6.316V101l-3.562.004-3.564.002-3.65 3.623-3.652 3.621H37.785 15v-3.625c0-2.752-.076-3.625-.316-3.625zm2.566-12.191V71.368l3.191-3.184L23.632 65h16.12 16.12L49.5 71.375l-6.371 6.375h-11.19H20.75V88v10.25h17.125H55V88 77.75h-.79c-.703 0-.056-.583 5.813-5.244l6.603-5.244.064 16.176.064 16.176-3.311 3.318-3.311 3.318H38.691 17.25zm55-25.912c0-.057 2.081-1.765 4.625-3.795L81.5 55.41v-1.705V52H75c-3.575 0-6.5-.091-6.5-.203s.316-.506.701-.875l.701-.672h18.365 18.365l3.619-3.625L113.869 43h3.566H121v6.316 6.316l-3.691 3.684-3.69 3.684H92.934c-11.376 0-20.684-.046-20.684-.103zm-.375-14.716c0-.038 1.73-1.864 3.844-4.059l3.844-3.99 10.656-.004 10.656-.004V29.938 19.75H83.813 66.75v10.182 10.182l.844.037.844.037-2.625 2.029-2.625 2.029-.032-15.403-.032-15.403 3.218-3.22L69.561 7h21.595 21.595v17.407 17.407l-3.22 3.218-3.22 3.218H89.093c-9.47 0-17.218-.031-17.218-.069zm-18.842 2.95v-.841l3.67-3.667c2.63-2.627 3.784-3.667 4.073-3.667.391 0 .402.043.352 1.45l-.051 1.45-3.889 3.048-4.022 3.057c-.073.005-.133-.369-.133-.831zm0 11.094v-.884h3.308 3.308l-.871.884-.871.884H55.47h-2.437z"/></svg>

```

File: devicon.json [hunk 6]
```diff
@@ -3151,6 +3151,26 @@
...
+        "name": "putty",
+        "tags": [
+            "ssh",
+            "server"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+				"plain",
+				"line"
```
Comment (Panquesito7) [inline devicon.json:3165]:
```suggestion
                "original",
                "plain",
                "line"
```

File: devicon.json [hunk 7]
```diff
@@ -3151,6 +3151,26 @@
...
+            ],
+            "font": [
+				"plain",
+                "line"
```
Comment (Panquesito7) [inline devicon.json:3169]:
```suggestion
            "font": [
                "plain",
                "line"
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Wow, great work! 😄

**Comment by Panquesito7** [comment]
Almost there! You just have to make the thunderbolt a bit more visible in the `line` version. 🙂

**Comment by MichielPater** [comment]
> Almost there! You just have to make the thunderbolt a bit more visible in the `line` version. 🙂

Thank you! I wonder why it is not visible in the font, while I can see it in my SVG. I'm not sure how to fix it if I cannot reproduce it. What do you think?

**Review by Panquesito7 — COMMENTED** [review]
I have no idea why the `line` version is looking like that. Any ideas, @Thomas-Boi? Thanks.

**Comment by Thomas-Boi** [comment]
@Panquesito7 it is quite strange. This is also an issue in the other PRs as well. Checking it manually on Icomoon yield no issue. Sometimes, perhaps the SVG just can't be made into an icon.

Btw, @MichielPater, what software do you use to make these SVGs? Just curious.

Luckily, the `plain` version looks great 👍. I think we can remove the `line` version and just keep the `plain` one.

**Comment by MichielPater** [comment]
> @Panquesito7 it is quite strange. This is also an issue in the other PRs as well. Checking it manually on Icomoon yield no issue. Sometimes, perhaps the SVG just can't be made into an icon.
> 
> Btw, @MichielPater, what software do you use to make these SVGs? Just curious.
> 
> Luckily, the `plain` version looks great 👍. I think we can remove the `line` version and just keep the `plain` one.

Hi @Thomas-Boi I am using Inkscape

**Review by Panquesito7 — APPROVED** [review]
Amazing work, @MichielPater! Thank you for your patience, dedication, and contributions to our community! 😄👍🎉

**Review by Thomas-Boi — APPROVED** [review]
Hi @MichielPater

This actually looks way better than I expected. Good job on fixing the SVG 👍. Sorry for the late reply, I've been occupied with other tasks.

Thanks for your work and your SVGs look good. I'll merge your PR now.

**Pull request merged**